### PR TITLE
Update Sathsara.html

### PR DESCRIPTION
**Pull request by sathsara-7788**
i have change brach

### CODE REVIEW

File: src/profiles/Sathsara.html [hunk 1]
```diff
@@ -2,4 +2,13 @@
   <div>
     I am Tharusha sathsara
   </div>
+<div>
+   <ul>
+   <li>pascrd o/l exam </li>
```
Comment (PraAnj) [inline src/profiles/Sathsara.html]: Fix spelling issues

File: src/profiles/Sathsara.html [hunk 2]
```diff
@@ -2,4 +2,13 @@
...
+   <li>science c </li>
+   <li> sinhala c </li>
+     <li>dram a </li>
```
Comment (PraAnj) [inline src/profiles/Sathsara.html]: Remove space in drama

Comment (sathsara-7788) [reply src/profiles/Sathsara.html]: fixed

File: src/profiles/Sathsara.html (hunk 1 above)
Comment (sathsara-7788) [reply src/profiles/Sathsara.html]: fixed

**Review by PraAnj — APPROVED** [review]

**Pull request merged**